db/migrate/20161215153144_add_default_value_to_sponsors_featured.rb

```diff
@@ -0,0 +1,9 @@
+class AddDefaultValueToSponsorsFeatured < ActiveRecord::Migration
+  def up
+    change_column_default :sponsors, :featured, true
```

Comment:
This change_column_default only applies to new entries. So need to update via rails console in the server for existing entries.